x/ethbridge/handler.go

```diff
@@ -261,3 +256,40 @@ func handleMsgUpdateWhiteListValidator(
 
 	return &sdk.Result{Events: ctx.EventManager().Events()}, nil
 }
+
+func handleMsgUpdateCethReceiverAccount(
+	ctx sdk.Context, cdc *codec.Codec, accountKeeper types.AccountKeeper,
+	bridgeKeeper Keeper, msg MsgUpdateCethReceiverAccount, sugaredLogger *zap.SugaredLogger,
+) (*sdk.Result, error) {
+	account := accountKeeper.GetAccount(ctx, msg.CosmosSender)
+	if account == nil {
+		sugaredLogger.Errorw("account is nil.", "CosmosSender", msg.CosmosSender.String())
```

Comment:
Please remove all fields from this log that do not have a corresponding key. We want all logs to be structured JSON, not unstructured text.